pt-BR/mail/chrome/messenger/preferences/general.dtd

```diff
@@ -37,31 +37,31 @@
 
 <!-- message list font size -->
 
-<!ENTITY pbChromeFontSizeSmall.label      "Small">
-<!ENTITY pbChromeFontSizeMedium.label     "Medium">
-<!ENTITY pbChromeFontSizeLarge.label      "Large">
-<!ENTITY pbChromeFontSize.label           "Message List and Folder Font Size:">
+<!ENTITY pbChromeFontSizeSmall.label      "Pequeno">
+<!ENTITY pbChromeFontSizeMedium.label     "Médio">
+<!ENTITY pbChromeFontSizeLarge.label      "Grande">
+<!ENTITY pbChromeFontSize.label           "Tamanho da fonte da lista de mensagens e da pasta:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbChromeFontSize.label           "Tamanho da fonte para a lista de mensagens e pastas:">